addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -459,6 +459,35 @@ def test_assign_quota(self):
             "Assignment quota: no compensation and no limit anymore (do as asked even with 30 already assigned)"
         )
 
+    def test_random_choice(self):
```

Comment:
Test should be ordered alphabetically, so that they execute in the order of the file.
I would also rename it to something more specific like `test_assign_all_members_unasignable` (feel free to take some creative liberties ^^)

Reply:
Done!